tools/interactive/interactivetool_cellxgene_1.3.0.xml

```diff
@@ -0,0 +1,117 @@
+<tool id="interactive_tool_cellxgene" tool_type="interactive" name="Interactive CELLxGENE Environment" version="1.3.0+galaxy0">
+    <requirements>
+	    <container type="docker">quay.io/bgruening/cellxgene:1.3.0-0</container>
+    </requirements>
+    <entry_points>
+        <entry_point name="CELLxGENE Single Cell Visualisation on $infile.display_name" requires_domain="True">
+            <port>80</port>
+        </entry_point>
+    </entry_points>
+    <stdio>
+		<regex match="WARNING" source="both" level="warning" description="cellxgene warnings"/>
```

Comment:
Indentation looks wrong for this file? Are you sure you are using spaces and NOT tabs?